server/game/cards/06-WoE/MyxTheTallminded.js

```diff
@@ -0,0 +1,22 @@
+const Card = require('../../Card.js');
+
+class MyxTheTallminded extends Card {
+    // Your opponent's keys cost +1 Aember for each friendly Mars
+    // creature in play.
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            targetController: 'opponent',
+            effect: ability.effects.modifyKeyCost(
+                (player) =>
+                    (player.opponent &&
```

Comment:
What happens if opponent takes control of Myx?
I wonder if "player" here is who owns or who controls the source.

Reply:
This is all copied from EDAI so it should work fine, but I can add a test.

Reply:
No need,  I tried to find a reference but I could not. Hoefully EDAI also works when taken control